### PR TITLE
Pretty register page

### PR DESCRIPTION
**Pull request by tenisakb**
clipboard - february 15 2019 2-23 pm

### CODE REVIEW

File: src/components/register-record-recovery-phrase/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { connect } from "react-redux";
+
+import RegisterRecordRecoveryPhraseSlide from "./register-record-recovery-phrase-slide";
+
+const mapStateToProps = state => ({});
+
+const mapDispatchToProps = dispatch => ({});
+
+const RegisterRecordRecoveryPhrase = () => {
```
Comment (EdmundMai) [inline src/components/register-record-recovery-phrase/index.tsx]: `const RegisterRecordRecoveryPhrase = () => <RegisterRecordRecoveryPhraseSlide />;
`

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+const Content = styled.p`
+  width: auto;
+  font-size: 12px;
+  font-weight: ${props => props.theme.fontWeight};
+  font-style: ${props => props.theme.fontStyle};
+  font-stretch: ${props => props.theme.fontStretch};
+  line-height: ${props => props.theme.lineHeight};
+  letter-spacing: ${props => props.theme.letterSpacing};
+  color: ${props => props.theme.container.content};
+  margin: 0 30px 0 30px;
```
Comment (EdmundMai) [inline src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: 0 30px

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  padding-bottom: 40px;
+  text-align: right;
+  @media (max-width: ${DESKTOP_WIDTH}px) {
+    text-align: center;
+    margin: 40px;
+    padding: 10px;
+  }
+`;
+
+const FlexGrid = styled.div`
```
Comment (EdmundMai) [inline src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: rename to something meaningful

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 60px 0px 60px;
+`;
+
+const FlexCol = styled.div`
```
Comment (EdmundMai) [inline src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: rename to something meaningful

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+  width: 33%;
```
Comment (EdmundMai) [inline src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: is there a way we can avoid these % widths?

**Comment by EdmundMai** [comment]
what is with the background color?

pretty_register_page_by_tenisakb_ _pull_request__87_ _opacity_webinterface

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx (hunk 2 above)
Comment (tenisakb) [reply src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: fixed

File: src/components/register-record-recovery-phrase/index.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/register-record-recovery-phrase/index.tsx]: fixed

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx (hunk 3 above)
Comment (tenisakb) [reply src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: fixed

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx (hunk 4 above)
Comment (tenisakb) [reply src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: fixed

File: src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx (hunk 5 above)
Comment (tenisakb) [reply src/components/register-record-recovery-phrase/register-record-recovery-phrase-slide.tsx]: fixed

**Pull request merged**